e2e/__snapshots__/e2e.spec.ts.snap

```diff
@@ -545,7 +545,7 @@ bcr_test_module:
 modules/zip/1.0.0/source.json
 ----------------------------------------------------
 {
-    \\"integrity\\": \\"sha256-Bqu+8wSwNVkplPEHcMNdla+oQRGoyXfx4BohYHCPZFU=\\",
+    \\"integrity\\": \\"sha256-fencLRegfGaNYdAWP/WaXxUWJwzD19XQtpWf2qcmkZw=\\",
```

Comment:
Do we need to document which environment this represents?

Reply:
I haven't quite figured it out, except that the zips produces on my mac laptop match the ones produces on CI (ubuntu), but do not match the ones produced on my ubuntu laptop :flushed:. This is only test code though, nothing used in production.